DigitalLearningSolutions.Data.Tests/Services/TrackerServiceTests.cs

```diff
@@ -50,19 +52,55 @@ public void ProcessQuery_with_unknown_action_returns_InvalidAction_response()
         }
 
         [Test]
-        public void
-            ProcessQuery_with_GetObjectiveArray_action_passes_query_params_and_returns_appropriate_service_response()
+        public void ProcessQuery_with_GetObjectiveArray_action_passes_query_params()
         {
             // Given
             var query = new TrackerEndpointQueryParams
                 { Action = "GetObjectiveArray", CustomisationId = 1, SectionId = 1 };

```

Comment:
If you make the two query params different you can test they're being passed in the right order.